prime-router/src/test/kotlin/fhirengine/engine/FhirConverterTests.kt

```diff
@@ -294,6 +294,10 @@ class FhirConverterTests {
     fun `test getTransformerFromSchema`() {
         val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.process) as FHIRConverter)
 
+        mockkClass(BlobAccess::class)
```

Comment:
Why are these changes needed now?

Reply:
Because we are passing the blob connection info around now and so the things that use those methods need that call mocked otherwise they fail.